DSL/Ruuter/POST/common-services.yml

```diff
@@ -0,0 +1,37 @@
+declaration:
+  call: declare
+  version: 0.1
+  description: "Decription placeholder for 'COMMON SERVICES'"
+  method: get
+  accepts: json
+  returns: json
+  namespace: service
+
```

Comment:
Please add the missing `allowlist` with the used body fields to the `declaration` step, like so:
```
declaration:
  ...
  allowlist:
    body:
      - field: page_size
        type: number
        description: "Body field 'page_size'"
      - field: sorting
        type: string
        description: "Body field 'sorting'"
      ...etc
```